src/components/Sections/GoogleMapsSection/__tests__/GoogleMapsSection.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { render } from "@testing-library/vue";
+import "@testing-library/jest-dom";
+import GoogleMapsSection from "..";
+
+const apikey = process.env.VUE_APP_MAPS_APIKEY;
+
+const locations = [
+  {
+    position: {
+      lat: 48.214039,
+      lng: 16.346442,
+    },
+    name: "Skoda",
+    city: "Wien",
+    street: "Skodagasse 13",
+  },
+];
+
+describe("components/sections/google-maps-section", () => {
```

Comment:
* We should add tests for error cases (e.g. a wrong or missing api key).
* We should try to test all properties of the GoogleMapsSection (e.g. handleButtonClick, locations, ...)